docs/internal/maintenance/ReleaseProcess-create-test-distribution.md

```diff
@@ -122,14 +122,18 @@ If you already have the repository:
 Build the distribution zip file:
 
 1. `npm install` 
-2. `npm run release`
+2. `npm run build`
+3. `npm run release`
```

Comment:
`npm run release` is defined as `npm run build && node scripts/create_release.js`, so the `npm run build` command does not need to be run separately.